.npmignore

```diff
@@ -1 +1,10 @@
-logo
+.editorconfig
+.eslintcache
+.eslintignore
+.eslintrc.json
+.github
+.lintstagedrc
+assets
```

Comment:
Does assets imply they're used by Tapestry itself? My gut instinct with assets is that the server/app serves them to end users.